misc/metrics/src/swarm.rs

```diff
@@ -123,6 +127,15 @@ impl Metrics {
             Box::new(connections_closed.clone()),
         );
 
+        let connection_establishment_duration = Family::new_with_constructor(
+            create_connection_establishment_duration_histogram as fn() -> Histogram,
+        );
+        sub_registry.register(
+            "connection_establishment_duration",
+            "Time it took (locally) to finish establishing connections",
```

Comment:
```suggestion
            "Time it took (locally) to establish connections",
```

I think this is easier to read. I think the "locally" stresses enough that this is the local view only. Always hard to pick the perfect description.